ui/tests/acceptance/config-ui/login-settings-test.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { click, visit, currentRouteName } from '@ember/test-helpers';
+import { login } from 'vault/tests/helpers/auth/auth-helpers';
+import { GENERAL } from 'vault/tests/helpers/general-selectors';
+import { runCmd } from 'vault/tests/helpers/commands';
+
+module('Acceptance | Enterprise | config-ui/login-settings', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function () {
+    const version = this.owner.lookup('service:version');
+    version.type = 'enterprise';
+    await login();
+  });
+
+  test('fetched login rule list renders', async function (assert) {
+    await runCmd([
+      'write sys/config/ui/login/default-auth/testRule backupAuthTypes=[] defaultAuthType=okta disableInheritance=false namespace=ns1',
+      'write sys/config/ui/login/default-auth/testRule2 backupAuthTypes=[] defaultAuthType=ldap disableInheritance=false namespace=ns2',
+    ]);
```

Comment:
i'm kind of surprised these are camel cased and not snake cased 🤔 i thought the api service was transforming them (when we were testing last week). Since this is mimicking CLI commands I'm pretty sure these should follow the API write syntax which is snake cased, unless the web console does something strange